applications/machine_learning/README.rst

```diff
@@ -79,6 +79,31 @@ By default, the application uses pretrained machine leaning models deployed in `
 The application displays LED effects that correspond to the machine learning results.
 For more detailed information, see the `User interface`_ section.
 
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+In the Machine Learning application specified modules can be suspended or turned off if they are not in use.
+The application uses :ref:`caf_power_manager` module for that purpose.
+It means that Zephyr power management is used when device is in active on suspended state,
```

Comment:
`is not used` :)

Reply:
From my perspective it is - in that state it is OS that decides if we should put the CPU down.

Reply:
During module initialization we use `pm_power_state_force((struct pm_state_info){PM_STATE_ACTIVE, 0, 0});`. In that case when device is active/suspended, the Zephyr power management is actually disabled and not used since we force the active state. We change the forced state only on system off.

Reply:
btw. `active or suspended state` ?

Reply:
```
The application uses :ref:`caf_power_manager` module for this purpose, which means that Zephyr power management is forced to :c:enumerator:`PM_STATE_ACTIVE` when device is in either the active or the suspended state, but the power off state is forced directly by :ref:`caf_power_manager` as Zephyr's :c:enumerator:`PM_STATE_SOFT_OFF`.
```